StudCity.Backend/StudCity.API/Controllers/UserController.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using StudCity.API.ViewModels;
+using StudCity.Core.DTOs;
+using StudCity.Core.Exceptions;
+using StudCity.Core.Interfaces;
+
+namespace StudCity.API.Controllers;
+
+/// <summary>
+/// User API controller.
+/// </summary>
+[ApiController]
+[Route("/user")]
+public class UserController : ControllerBase
+{
+    private readonly IUserService _userService;
+
+    public UserController(IUserService userService)
+    {
+        _userService = userService;
+    }
+
+    /// <summary>
+    /// Update user information.
+    /// </summary>
+    /// <param name="user">Model that contains user's data.</param>
+    /// <response code ='204'>When informations updated successfully.</response>
+    [Authorize]
+    [HttpPut("update-info")]
```

Comment:
While we have an empty API controller, we can write a real RESTful controller.
That is: 


Method | Route | Description
-- | -- | --
GET | /user | get something...
POST | /user | do something...
PUT |  /user | Update user
DELETE | /user/{id} | delete user...

Reply:
i want to extend this controller and take the next step in implementing that service..
however i'm waiting for little bit of planning and guidance from you 
thanks a lot, wish you a great day 😀